### ISSUE TITLE
Dump all generated images to central directory

### ISSUE THREAD

**Issue by tir38** [arunkumar9t2/scabbard#23]
We want to bundle the generated .png files with our internal documentation. It would be helpful if they were all in a single director, w/out having to navigate through the project to find and move them.

**Comment by arunkumar9t2** [comment]
Agreed, I tried to get this done as part of `0.1.0` but could not make it. Rough [implementation](https://github.com/arunkumar9t2/scabbard/commit/07ada58a8b25497e7590eaaf98cfd3efc06e1ddf) which copies all scabbard files to `build/scabbard/` folder.

I'll create a separate issue to discuss the implementation details and link this there.

**Comment by tir38** [comment]
oh yeah, no rush. I love what you have so far!
